contextMenu.js

```diff
@@ -135,6 +136,26 @@
                             }
                         }
 
+						//update ico
+                        if (ico) {
+                            var imgIcon = elm.find('.iw-mIcon');
+                            if (imgIcon.length) {
+								if(imgIcon[0].tagName.toLowerCase() === "span"){
```

Comment:
As its a jquery dependent library. You can use .is() method. ie imgIcon.is('span')